### ISSUE TITLE
Slider doesn't slide smoothly on setting smooth flag to true

### ISSUE THREAD

**Issue by samdanishaik** [darul75/angular-awesome-slider#130]
Looks like the slider only listens to the drag event and doesnt handle the swipe event. It would be better if swipe event is also handled to make the slider smooth transition on the user interface.

**Comment by darul75** [comment]
hi @samdanishaik,

By default smooth option is set to true... and make slider more reactive, when set to false it rounds the pixel position value.

Today for mobile, used events are touchstart, touchend..., I do not know about how to deal with swipe events but it looks like they are triggered after a consequent amount of pixels touch...

**Comment by samdanishaik** [comment]
Yes, Julien that is exactly is the issue. I am just wondering how to improve it. Will let you know if I get any suggestions. Meanwhile you also try addressing it if you have any solution in your mind. Thanks.
